packages/fast-components-styles-msft/src/progress/index.ts

```diff
@@ -139,59 +140,59 @@ const styles: ComponentStyles<ProgressClassNameContract, DesignSystem> = {
     },
     progress_dot__1: {
         width: "40%",
-        transform: "translateX(-100%)",
+        // transform: directionSwitch("translateX(-100%)", "translateX(100%)"),

```

Comment:
Should this be commented out? (Same with line 148

Reply:
Obviously not, sorry!